wpeverest/user-registration
# UR-1335 Tweak - From address going wrong in some emails

### All Submissions:



### Changes proposed in this Pull Request:

Previously, the from address while sending the email address was not working properly. This PR will fix this issue. Now, We have also added the additional email setting for Confirm Email Address Change to modify the contents and subjects for Confirming the email address to the new user's email address. If you want to test it with the email template the please pull  https://github.com/wpeverest/user-registration-email-templates/tree/UR-1335-tweak/from-address-going-wrong-in-some-emails branch on user-registration-email-templates.
Closes # .

### How to test the changes in this Pull Request:

1. Switch to this branch 
2. Pull https://github.com/wpeverest/user-registration-email-templates/tree/UR-1335-tweak/from-address-going-wrong-in-some-emails for updated email templates
3. Go to UR>Settings>Emails
4. Change the From Address
5. Click on Send Email to test the email for verifying the from email address.
6. Enable **Confirm Email Address Change toggle** to send message with email change link in the email address.
7. Also, verify it according to the email template. 

### Types of changes:



### Other information:



### Changelog entry

> UR-1335 Tweak - From address going wrong in some emails
